src/main/java/org/jabref/preferences/PreferencesService.java

```diff
@@ -70,6 +70,8 @@
 
     void setWorkingDir(Path dir);
 
+    String setLastPreferencesExportPath();
```

Comment:
Is this still needed?

Reply:
Partly. I would like to postpone refactor of this until later when I'm working on FilePreferences.